### ISSUE TITLE
Qt 6.2 Alpha now released

### ISSUE THREAD

**Issue by clin1234** [Coding-Bunker/unofficial-discord-client#55]
https://www.qt.io/blog/qt-6.2-alpha-released. This means Qt Websockets and Qt Multimedia are available (only on Linux)

**Comment by clin1234** [comment]
When Qt 6.2 is stable, bump minimum required Qt version for Qt Multimedia?

**Comment by guerinoni** [comment]
Yes but you are free to implement with other socket libraries... All feature are welcome :)

**Comment by Tachi107** [comment]
Qt 6.2 has been stable for a while now, are you still interested in developing the voice chat feature?

**Comment by clin1234** [comment]
Far as I can tell, aside from DependBot notifications, there's little development effort happening.

If you want to start it, go nuts

**Comment by Tachi107** [comment]
I'd really like to, I've been wanting to learn Qt for quite some time, but I'm busy with other projects right now and I don't care enough about Discord to spend time on a Discord client :)

**Comment by guerinoni** [comment]
You are free to implement it! I will leave a review :D 
Hope someone can take this project :)
